### DISCUSSION TITLE
CPS1

### DISCUSSION THREAD

**Discussion by vics3g** [OnionUI/Onion#520] [General]
What is happening with these CORE? Games like AVP now are not working with last update? Anyone know why work really really bad and most of games doesn't work?

**Comment by dwmccoy** [comment]
More than likely a RomSet issue.  I have various FBALPHA/FBNEO Romsets.  The Alien vs. Predator I have from RomSet 0.297.39 will not work with the CPS2 default emulator but that same rom works fine with the FBNeo emulator from the expert menu.  I found that my AVP rom from FBAlpha 0.2.97.44 Romset plays just fine with the default CPS2 emulator.  If I'm not mistaken, the 0.2.97.44 was the last Romset released for FBAPLHA.  Romsets after that were released for FBNeo.  My suggestion would be to try the roms from 0.2.97.44
